pipelines/folioRancher/folioNamespaceTools/manageNamespace/manageSecrets/Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+#!groovy
+import groovy.json.JsonOutput
+import org.folio.Constants
+import org.folio.rest.GitHubUtility
+import org.folio.utilities.Tools
+import org.jenkinsci.plugins.workflow.libs.Library
+
+LIB_BRANCH = 'RANCHER-741'
+@Library('pipelines-shared-library@RANCHER-741') _
```

Comment:
Do not forget to change branch here to RANCHER-741-Jenkins-Enhancements before merge

Reply:
Full branch name should be